docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,73 @@
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes in three availability zones and deploy at least three pods. You also should ensure that one pod is deployed on one worker node by using [Node Affinity](https://kubernetes.io/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/).
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDB Cluster are limited to 2vCPU / 4GB memory. In addition, we recommend deploying one ScalarDB Cluster pod and one Envoy pod on one worker node. Note that you do not need to deploy an Envoy pod when using `direct-kubernetes` mode.
+
+In other words, the following components run on one worker node:
+
+* ScalarDB Cluster pod (2vCPU / 4GB)
+* Envoy proxy (0.2–0.3 vCPU / 256–328 MB)
+* Kubernetes components
+
+With this in mind, you should use a worker node that has 4vCPU / 8GB memory resources. We recommend running only the above components on the worker node for ScalarDB Cluster.
+
+### Network
+
+You should create the Kubernetes cluster on a private network since ScalarDB Cluster does not provide any services to users directly via internet access. We recommend accessing ScalarDB Cluster via a private network from your applications.
+
+### Monitoring and logging
+
+You should monitor the deployed components and collect their logs. For details, see [Monitoring Scalar products on a Kubernetes cluster](./K8sMonitorGuide.md) and [Collecting logs from Scalar products on a Kubernetes cluster](./K8sLogCollectionGuide.md).
+
+### Backup and restore
+
+You should enable the automatic backup feature and PITR feature in the backend database. For details, see [Set up a database for ScalarDB/ScalarDL deployment](./SetupDatabase.md).
+
+## Production checklist: Client application that uses ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up a client application that is using ScalarDB Cluster in a production environment.
+
+### Transaction manager configuration
+
+You must always run CRUD requests via ScalarDB Cluster. To ensure requests are running properly, check the properties file for your client application and confirm that `scalar.db.transaction_manager=cluster` is configured.
+
+* Recommended for production environments
+  ```mermaid
+  flowchart LR
+    A(<b>App</b> <br> ScalarDB Cluster Library with gRPC) --> B(<b>ScalarDB Cluster</b> <br> ScalarDB Library with Consensus Commit) --> C(Underlying storage/database)
+  ```
+
+* Not recommended for production environments (for testing purposes only)
+  ```mermaid
+  flowchart LR
+    A(<b>App</b> <br> ScalarDB Cluster Library with Consensus Commit) --> B(Underlying storage/database)
+  ```
+
+### Deployment of the client application when using `direct-kubernetes` client mode
```

Comment:
Do we first need to recommend users to use the direct mode whenever possible?

Reply:
Ah, yes. That's right. We should do so.
I will add a new section like `Use direct mode if you can`.

Reply:
I added the new section `Client mode` in dcd5987f543ad6668b1aee69f51f1c50f1c31a3f.